spec/controllers/thredded/topics_controller_spec.rb

```diff
@@ -69,5 +69,32 @@ module Thredded
         expect(assigns(:new_topic).content).to eq 'preset content'
       end
     end
+
+    describe 'POST follow' do
+      subject { post :follow, messageboard_id: @messageboard.id, id: @topic.id }
+      it 'follows' do
+        expect { subject }.to change { @topic.reload.followers.reload.count }.by(1)
+      end
+      context 'json format'
```

Comment:
oops! yup, that wasn't actually segregating the context (context without a block is a noop). will fix